jest-lint.config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "runner": "jest-runner-eslint",
+  "displayName": "lint",
+  "moduleFileExtensions": ["ts", "js"],
+  "transform": {
+      "^.+\\.(ts|js)$": "typescript-babel-jest"
+  },
+  "testMatch": ["**/src/**/*.(ts|js)"]
```

Comment:
might be more trouble than its worth, but we could use .js extensions for the `jest*config` files and extend a base one that has moduleFileExtensions, transform and testMatch in it. 

what do you think?

Reply:
+1 to that